great-idea/css/index.css

```diff
@@ -63,4 +63,130 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+
+#container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+#navwrapper {
+	display: flex;
+}
+
+nav {
+	margin-top: 35px;
+	display: flex;
+}
+
+nav > a {
+	display: flex;
+	color: gray;
+	text-decoration: none;
+	font-size: .9em;
+	margin-right: 80px;
+}
+
+.logo {
+	margin-top: 20px;
+}
+
+nav > a:hover {
+	color: royalblue;
+	border-bottom: 1px solid royalblue;
+}
+
+h1 {
+	font-size: 4.5em;
+	width: 250px;
+	margin-top: 45px;
+	margin-left: 55px;
+	letter-spacing: 2px;
+	/* text-align: center; */
+}
+
+#banner {
+	display: flex;
```

Comment:
try to use more specificity in your project, this will help later on when you start building apps with muliple pages that use the same css file. You could end up with a header or banner in the second page that you don't want to share styles. If you use something like .container .content .banner { <--- this will make sure that nothing else with the class of banner can be affected by this style because it's so specific that css can't mistake it.